lambda/src/main/scala/pricemigrationengine/model/Failure.scala

```diff
@@ -23,3 +23,4 @@ case class S3Failure(reason: String) extends Failure
 case class SubscriptionIdUploadFailure(reason: String) extends Failure
 
 case class NotificationEmailFailure(reason: String) extends Failure
+case class EmailSenderFailure(reason: String) extends Failure
```

Comment:
What's the difference between these?

Reply:
Oops

Reply:
Ah, actually they are different, one is raised by the EmailSender service one by the NotificationEmailHandler.

Ill leave it as is.